### ISSUE TITLE
Codecov reports failure on `master`

### ISSUE THREAD

**Issue by spawnia** [webonyx/graphql-php#910]
@simPod Codecov reports a failed pipeline on `master`: 
![image](https://user-images.githubusercontent.com/12158000/128017271-404d2e2f-90b0-4274-98cc-1cb2f5a9389e.png)

Can we turn that off and only have it fail pull requests?

**Comment by simPod** [comment]
Yup, the patch coverage is lower than expected. I'll check what to do about that.
